pyproject.toml

```
@@ -17,7 +17,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.8.1"
-django = ">=4.0,<=4.2"
+django = ">=4.0,<=4.3"
```

There's no Django 4.3 so I'd suggest this change.

```suggestion
django = ">=4.0,<5.0"
```